isos/bootstrap/systemd-init

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# This entry point gets invoked invoked for systemd based
+# bootstraps.
+# It will load any custom prep for the repo in question, as
+# defined in repos/name/init.sh then continue with the common
+# bootstrap script
+
+# path consistency
+export PATH=$PATH:/sbin
+
+# load the customizations
+echo "######## Loading repo customizations ########"
+. /bin/repoinit
+echo "######## Done repo customizations ######## -- exit $?"
+
+mount -t proc none /proc
+mount -t sysfs none /sys
+mount -t devtmpfs none /dev
+
+# make some of the basics and start udev
+mknod -m 622 /dev/console c 5 1
+mknod -m 666 /dev/null c 1 3
+mknod -m 666 /dev/zero c 1 5
+mknod -m 666 /dev/ptmx c 5 2
+mknod -m 666 /dev/tty c 5 0
+mknod -m 444 /dev/random c 1 8
+mknod -m 444 /dev/urandom c 1 9
+chown root:tty /dev/{console,ptmx,tty}
+
+mkdir -p /dev/{pts,shm}
+mount -t devpts -o gid=4,mode=620 none /dev/pts
+mount -t tmpfs none /dev/shm
+
+/usr/lib/systemd/systemd-udevd --daemon
```

Comment:
We also need to ensure, both this and sysv, that the cgroups hierarchy is constructed.